AngouriMath/Core/Domains.cs

```diff
@@ -47,10 +47,10 @@ public static bool FitsDomainOrNonNumeric(Entity entity, Domain domain)
             (Domain.Any, "AA")
```

Comment:
Universal set (UU)?

Reply:
Universal set is specific some area. Say, in math we usually refer to U as to the whole set of reals, complexes (depends on the context). Here it means literally anything 🦆

Reply:
https://en.wikipedia.org/wiki/Universal_set
> In set theory, a universal set is a set which contains all objects, including itself.

It really just depends on what you can work with.